---
title: "buttRflyfish: reproduction of figure 2b from Nowicki et al. 2020"
author:
  - Gabriel A. Preising:
      email: gpreisin@stanford.edu
      institute: UofO
      correspondence: true
  - Tatiana Bellagio:
      email: tbellg@stanford.edu
      institute: UofO
      correspondence: false
  - Sebastian ToroArana:
      email: sebatoro@stanford.edu
      institute: UofO
      correspondence: true  
institute:
  - UofO: Stanford University Biology Department
date: "`r format(Sys.time(), '%d %B, %Y')`"
output: 
    bookdown::word_document2:
      fig_caption: yes
      reference_docx: "../templates/template.docx" # Insert path for the DOCX file
      pandoc_args:
      - --lua-filter=../templates/scholarly-metadata.lua
      - --lua-filter=../templates/author-info-blocks.lua
      - --lua-filter=../templates/pagebreak.lua
bibliography: references.bib
csl: "../templates/journal-of-archaeological-science.csl" # Insert path for the bib-style
---

```{r setup, echo = FALSE}
knitr::opts_chunk$set(
  collapse = TRUE,
  warning = FALSE,
  message = FALSE,
  echo = FALSE,
  comment = "#>",
  fig.path = "../figures/",
  dpi = 300
)

```

<!-- The actual document text starts here: -->

# Introduction
Here, we reproduce figure 2b from [@nowicki2020] - "Gene expression correlates of social evolution in coral reef butterfly fishes". This study analyzed the neural correlates of pair-bonding vs. solitary mating behavior in several species of *Chaetodontidae* fishes. To accomplish this, they compared gene expression of socially-relevant neuropeptides in brain regions pertinent to social function. One of their main findings was that the males of pair-bonding species had elevated oxytocin receptor (OTR) expression in the supracommissural part of the ventral telencephalon (Vs) compared to the males of solitary species.

# Methods
The authors of [@nowicki2020] used qRT-PCR to quantify relative mRNA expression of OTR in Vs of 5 *Chaetodontidae* species: *C. vagabundus*, *C. lunatus*, *C. baronessa*, *C. trifascialis*, and *C. rainfordi*. They employ a Markov Chain Monte Carlo approach to analyze differential gene expression. Below is a reproduction of figure 2b using our package `buttRflyfish`.

# Results

<!-- Here's some example analysis code: -->
```{r fig1, fig.cap="Reproduction of figure 2b from Nowicki et al. 2020.", echo = TRUE, results = 'hide', warning=FALSE, message = FALSE, fig.height=4, fig.width=4}
library(buttRflyfish)

data = load_data() # load data
model = run_model(data) # run MCMC model
p = otr_expression_plot(model,data) # plot differential expression results

```

# References

<!-- The following line ensures the references appear here for the MS Word or HTML output files, rather than right at the end of the document (this will not work for PDF files):  -->

::: {#refs}
:::

\newpage

### Colophon

This report was generated on `r Sys.time()` using the following computational environment and dependencies:

```{r colophon, cache = FALSE}
# which R packages and versions?
if ("devtools" %in% installed.packages()) devtools::session_info()
```

The current Git commit details are:

```{r}
# what commit is this file at? 
if ("git2r" %in% installed.packages() & git2r::in_repository(path = ".")) git2r::repository(here::here())  
```
